hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/keys/Ed25519Utils.java

```diff
@@ -128,6 +129,19 @@ public static File relocatedIfNotPresentWithCurrentPathPrefix(
         }
     }
 
+    private static String withDedupedHederaNodePathSegments(@NonNull final String loc) {
```

Comment:
Why do we need this ? 🤔

Reply:
sort of an answer at https://github.com/hashgraph/hedera-services/pull/8404#discussion_r1317397817